Tools/autotest/ardusub.py

```diff
@@ -266,6 +266,146 @@ def Surftrak(self):
         self.disarm_vehicle()
         self.context_pop()
 
+    def prepare_synthetic_seafloor_test(self, sea_floor_depth):
+        self.set_parameters({
```

Comment:
I played around some with delay, and I was able to get the tests to pass with a delay of 0.5s by setting additional parameters:

            "PSC_JERK_Z": 8,
            "PILOT_ACCEL_Z": 500,
            "WPNAV_ACCEL_Z": 500,

See https://github.com/clydemcqueen/ardusub_surftrak?#sensor-notes for details.